### DISCUSSION TITLE
Development is probably stopped here

### DISCUSSION THREAD

**Discussion by SebastJava** [SebastJava/mint-themes#57] [General]
Sorry for not making any follow up on your latest issues and requests. I am currently busy with other things.

These Mint-Yz themes and icons were a sort of "spin-off" after an aborted attempt to push better colors on the official [Mint-Y](https://github.com/linuxmint/mint-themes/tree/master/src/Mint-Y). Now that these colors finally got changed on the official Mint-Y version, after years of desperate waiting, these alternative versions here have lost some of their relevance. I still think some of my colors are better, but there isn't such a big difference anymore.

I am still currently using those Mint-Yz themes, of course. And i am happy to see many people still like it and use it.

Apart from having lost some of its relevance, there is another big reason to stop development here. I am looking at the [Materia theme](https://github.com/nana-4/materia-theme). Apart from the colors, there are many things i am still not satisfied with Mint-Y and Mint-Yz. This Materia theme looks much more interesting. But there are a few things i still don't like about this one. I want to modify it before starting using it. I am not even sure i will ever start working on it. First, i need to take a break. But if i was to go on and start working on GTK themes again, then i would probably forget about those Mint-Y themes and start a new fork from the Materia theme instead.

P.S.: This doesn't mean you should stop using these Mint-Yz themes right away. There isn't any security issue. You can just consider these themes to be "perfectly stable" :)

**Comment by fblais** [comment]
Un gros merci pour ton travail, Sébastien!
Il y a de nombreux thèmes dérivés de Materia. J'aime bien Vimix.
Bonne chance dans tes nouveaux projets!

**Reply by fblais** [reply]
Matcha est encore mieux, mais dérive de Arc, et je ne sais pas si c'est basé sur Material...

**Comment by SKEdmis** [comment]
Moi aussi je te remercie Sebastien. J'étais impressioné et content de trouver les thèmes Yz. Je les utilise toujours.

En espérant de voir un thème futur signé SebastJava...

Steve

**Comment by warthunder1969** [comment]
Sorry to hear you're stopping development, but glad you made the themes when you did :) I've been running them for years now and I do quite like them. I'll keep using them as it makes sense. If you do fork Materia themes I will take a look at whatever that is. Thanks for the time and effort you were able to put into these themes!
